src/read.c

```diff
@@ -3474,7 +3522,7 @@ static avifResult avifDecoderCreateCodecs(avifDecoder * decoder)
 // If alpha is AVIF_FALSE, searches for the primary color item (parentItemID is ignored in this case).
 // If alpha is AVIF_TRUE, searches for the auxiliary alpha item whose parent item ID is parentItemID.
 // Returns the target item if found, or NULL.
-static avifDecoderItem * avifDecoderDataFindItem(avifDecoderData * data, avifBool alpha, uint32_t parentItemID)
+static avifDecoderItem * avifDecoderDataFindItem(const avifDecoderData * data, avifBool alpha, uint32_t parentItemID)
```

Comment:
this change seems unrelated. please keep it out of this PR (it's already getting quite large).